okada_html/home.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>dobocreate株式会社</title>
+    <link rel="stylesheet" href="dobocreate.css" />
+    <style>
+        </style>
+  </head>
+  <body>
+    <header>
+            <div class="logo">
+                <img src="img/ドボクリエイト0-e1626942215397.jpg"alt="ロゴ" width=250px></div>
+                <ul>ホーム</ul> 
+                <ul>会社情報</ul> 
+                <ul>採用情報</ul> 
+                <ul>事業内容</ul> 
+                <ul>発表論文 </ul> 
+                <ul>アクセス</ul> 
+                <ul>お問い合わせ</ul> 
+                <ul>Instagram</ul> 
+    </header>
+    <section>
+        <div class="greeting">
+            <h1 >ごあいさつ</h1>
```

Comment:
不要なスペースを削除しておいてください。
```suggestion
            <h1>ごあいさつ</h1>
```